test/integration/conftest.py

```diff
@@ -100,3 +101,19 @@ def create_context(
         utils.close_environments(spawned)
 
     return create_context
+
+
+def exact_matcher(names):
+    return lambda value: all(x == value for x in names)
+
+
+def superset_matcher(names):
+    return lambda value: all(x in value for x in names)
+
+
+@contextlib.contextmanager
+def container_named(*names, matcher=None):
+    matcher = matcher if matcher else exact_matcher(names)
+    with ContextDockerClient() as client:
+        matches = [c for c in client.containers.list() if matcher(c.name)]
+        yield matches[0] if matches else None
```

Comment:
this can't stay in the conftest, after some research and also discussion with @Nicoretti , we determined that you should import the conftest and as such you can only have fixtures and their private help functions in the conftest

Reply:
The move of this function in the previous PR was intentional